doc/setup_instructions.md

```diff
@@ -338,39 +247,79 @@ If it didn't work you can try these:
 to get help. Links to the forums and community discord
 are at the top of this document.
 
-Forking
--------
+Git LFS limits
+--------------
 
-If you are just starting out and you don't have write access to the
-Thrive repository yet, you will need to create a fork. Use the fork
-button on Github on the main repository page. This will give you a
-fork under your github account (for example my fork is https://github.com/hhyyrylainen/Thrive).
+Due to current Git LFS limitations you can't commit changes to files tracked
+by Git LFS if you are not a team member. If that is needed for a PR please
+ask some team member to commit the assets for you.
 
```

Comment:
Linking to the wiki page for Git LFS would be nice here: https://wiki.revolutionarygamesstudio.com/wiki/Git_LFS